src/main/java/tp/cap5buddy/ui/Ui.java

```diff
@@ -0,0 +1,28 @@
+package tp.cap5buddy.ui;
+
+import tp.cap5buddy.parser.Parser;
+
+public class Ui {
+    private String userInput;
+    private String reply;
+
+    public Ui() {
+        this.userInput = null;
+        this.reply = null;
+    }
+
+    public void input(String input) {
```

Comment:
Maybe we could add an output method for whoever is doing the "View Module" portion.